scripts/linkbot/monitor-ub-server.js

```diff
@@ -33,14 +34,28 @@ JSDOM.fromURL(url, {}).then(dom => {
             return parseInt(b.groups.release) - parseInt(a.groups.release) || parseFloat(b.groups.minor) - parseFloat(a.groups.minor)
         })
     var matchOn = matches[0].input
-    if (matchOn !== imageName) {
-        console.log("We've got a new release! \\o/");
-        var newLine = ""
-        if (matches[0].groups.minor) {
-            newLine = `UBUNTU_SERVER_RPI_IMAGE_URL=\"${url}${matches[0].groups.release}.04.${matches[0].groups.minor}/release/ubuntu-${matches[0].groups.release}.04.${matches[0].groups.minor}-preinstalled-server-armhf+raspi.img.xz\"`
-        } else {
-            newLine = `UBUNTU_SERVER_RPI_IMAGE_URL=\"${url}${matches[0].groups.release}.04/release/ubuntu-${matches[0].groups.release}.04-preinstalled-server-armhf+raspi.img.xz\"`
+
+    return matchOn;
+
+    // Get the release image url from the releases (sub)-page
+    // const url = "http://cdimage.ubuntu.com/ubuntu/releases/"
+}).then(releaseVersion => {
+    var releaseVersion = releaseVersion.replace(/\s/g, "").replace(/\//g, "")
+    JSDOM.fromURL(url + releaseVersion + "/release/", {}).then(dom => {
```

Comment:
```suggestion
    JSDOM.fromURL(`${url}${releaseVersion}/release/`, {}).then(dom => {
```
just for consistency 😉